hub/core/storage/lru_cache.py

```diff
@@ -195,7 +222,10 @@ def get_bytes(
             if path in self.lru_sizes:
                 self.lru_sizes.move_to_end(path)  # refresh position for LRU
             return self.hub_objects[path].tobytes()[start_byte:end_byte]
-        elif path in self.lru_sizes:
+        elif path in self.lru_sizes and not (
```

Comment:
can you please comment which case it is trying to avoid?

Reply:
done